src/abstractdataframe/iteration.jl

```diff
@@ -38,21 +38,24 @@ end
 """
     DataFrameColumns{<:AbstractDataFrame, V} <: AbstractVector{V}
 
-Iterator over columns of an `AbstractDataFrame`.
-If `V` is `Pair{Symbol,AbstractVector}` (which is the case when calling
-[`eachcol`](@link)) then each returned value is a pair consisting of
-column name and column vector. If `V` is `AbstractVector` (a value returned by
-the [`columns`](@link) function) then each returned value is a column vector.
+Iterator over columns of an `AbstractDataFrame` constructed using
+[`eachcol(df, true)`](@link) if `V` is a `Pair{Symbol,AbstractVector}`. Then each
```

Comment:
```suggestion
[`eachcol(df, true)`](@ref) if `V` is a `Pair{Symbol,AbstractVector}`. Then each
```
Right?

Reply:
I will check with documenter - it was your change so I retained it and I never remember for sure what should be used when.

Reply:
You are right. Fixed.